GitLab/custom-executor.md

```diff
@@ -5,12 +5,39 @@ This guide explains how to set up a GitLab [Custom][custom] executor in a MacSta
 The Custom executor allows you to run builds on environments not supported natively by the GitLab [Runner][runner] (such as Orka). It also provides the flexibility to define how the environment should be set up and cleaned up.
 
 MacStadium provides Custom executor scripts that can be used to run CI/CD pipelines in an Orka environment.
+The Custom executor spins up ephemeral Mac machines, which will execute the CI/CD jobs.
 
 ## Requirements
 
 - GitLab [Runner][runner]
 - [jq][jq] - command-line JSON processor used by the provided scripts
 
+## Setup overview
+
+1. Set up an Orka VM base image. The image must have SSH enabled.
+2. Set up an Orka VM config using the base image from **Step 1**. The Orka VM config is the container template that Custom executor will use to spin up ephemeral Mac machines.
+3. Set up the GitLab Runner.
+
+## Set up an Orka VM base image
+
+The Orka VM used by the Custom executor to run CI/CD jobs.
```

Comment:
This is an incomplete sentence and it's meaning is lost. Can you rewrite it for clarity?